cpmpy/transformations/reification.py

```diff
@@ -77,7 +77,7 @@ def only_implies(constraints):
         # Comparisons: transform bV == BE
         elif cpm_expr.name == '==' and cpm_expr.args[0].is_bool():
             a0,a1 = cpm_expr.args
-            if isinstance(a0, _BoolVarImpl) and isinstance(a1, _BoolVarImpl):
+            if isinstance(a1, _BoolVarImpl):
```

Comment:
for completeness, maybe add  # assumption: a0 is a BoolVar
is it always? or did you derive it from the comparison?

Reply:
well the prerequisite is only_bv_reifies, so any reification should have a a bv there, will add comment